src/main/archetype/ui.config/src/main/content/jcr_root/apps/__appId__/config.author/com.day.cq.wcm.mobile.core.impl.MobileEmulatorProvider-__appId__.cfg.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Are you sure that cfg.json is supported by AEM 6.3? Or are you modifying the minimum system requirements for the upcoming archetype version?

Reply:
Since Core Components 2.9.0, we dropped support of AEM 6.3 as it reached the end of core support in May 2020 (see https://helpx.adobe.com/support/programs/eol-matrix.html). Hence, the Archetype 24 will also support only AEM 6.4+, meaning that we'll have to update the docs accordingly (mainly https://github.com/adobe/aem-project-archetype#system-requirements).

Reply:
Thanks for the feedback! I updated the readme accordingly and dropped the support for AEM 6.3